kpi-tracker/src/main.rs

```diff
@@ -27,22 +35,41 @@ struct Args {
     /// The node used for querying
     #[arg(
         long = "node",
-        help = "The endpoint is expected to point to a concordium node grpc v2 API.",
-        default_value = "http://localhost:20001"
+        help = "The endpoints are expected to point to concordium node grpc v2 API's.",
+        default_value = "http://localhost:20001",
+        env = "KPI-TRACKER-NODES"
     )]
-    node:          Endpoint,
+    node_endpoints: Vec<Endpoint>,
     /// Database connection string.
     #[arg(
         long = "db-connection",
         default_value = "host=localhost dbname=kpi-tracker user=postgres password=password \
                          port=5432",
         help = "A connection string detailing the connection to the database used by the \
-                application."
+                application.",
+        env = "KPI-TRACKER-DB-CONNECTION"
     )]
-    db_connection: tokio_postgres::config::Config,
+    db_connection:  tokio_postgres::config::Config,
     /// Logging level of the application
-    #[arg(long = "log-level", default_value_t = log::LevelFilter::Debug)]
-    log_level:     log::LevelFilter,
+    #[arg(long = "log-level", default_value_t = log::LevelFilter::Debug, env = "KPI-TRACKER-LOG-LEVEL")]
+    log_level:      log::LevelFilter,
+    /// Number of parallel queries to run against node
+    #[arg(
+        long = "num-parallel",
+        default_value_t = 1,
+        help = "The number of parallel queries to run against a node. Only relevant to set to \
+                something different than 1 when catching up.",
+        env = "KPI-TRACKER-NUM-PARALLEL"
+    )]
+    num_parallel:   u8,
+    /// Max amount of seconds a response from a node can fall behind before
+    /// trying another.
+    #[arg(
+        long = "max-behind-seconds",
+        default_value_t = 240,
+        env = "KPI-TRACKER-MAX-BEHIND-SECONDS"
+    )]
+    max_behind_s:   u8,
```

Comment:
Let's not use `u8`. That means on you allow at most 4min16s for delay :) 

Use `u32`